src/Chainweb/WebBlockHeaderDB.hs

```diff
@@ -100,7 +105,14 @@ webEntries db f = go (view (webBlockHeaderDb . to toList) db) mempty
     go [] s = f s
     go (h:t) s = entries h Nothing Nothing Nothing Nothing $ \x ->
         go t (() <$ S.mergeOn (view blockCreationTime) s x)
-            -- FIXME: should we include the rank in the order?
+
+-- | Returns all blocks in all block header databases.
```

Comment:
Add a comment why the block creation time is not working here. Maybe also add a comment about the order of blocks.